src/user-picker/UserPickerCard.js

```diff
@@ -37,6 +37,10 @@ type State = {
   selected: User[],
 };
 
+type HandleUserPressParams = {
+  email: string,
+};
```

Comment:
Let's inline types like this one. Like so:
```
handleUserPress = (params: { email: string }) => {
```
Otherwise they create clutter up here, and also make the actual function definition harder to read because the type that basically defines the signature is dozens of lines away up here.